src/main/java/com/cdancy/bitbucket/rest/domain/sshkey/AccessKey.java

```diff
@@ -17,16 +17,46 @@
 
 package com.cdancy.bitbucket.rest.domain.sshkey;
 
+import java.util.List;
+
 import com.google.auto.value.AutoValue;
+import org.jclouds.javax.annotation.Nullable;
 import org.jclouds.json.SerializedNames;
 
+import com.cdancy.bitbucket.rest.domain.common.Error;
```

Comment:
:warning: Extra separation in import group before 'com.cdancy.bitbucket.rest.domain.common.Error'
Wrong lexicographical order for 'com.cdancy.bitbucket.rest.domain.common.Error' import. Should be before 'org.jclouds.json.SerializedNames'. (#QR78:26#)-CHECKSTYLE_CustomImportOrderCheck